packages/kotti-ui/source/kotti-filter/components/FilterButtonLink.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+	<div class="kt-filter-button-link">
+		<span @click="handleClick">
+			<i class="yoco" v-text="icon" /> {{ label }}
+		</span>
+	</div>
+</template>
+
+<script lang="ts">
+import { defineComponent } from '@vue/composition-api'
+
+export default defineComponent<{ label: string; icon: string }>({
+	name: 'FilterButtonLink',
+	props: {
```

Comment:
Please deep sort and remove newlines, to align with the rest of the props in the codebase